src/components/LikeButton/docs/extendedExamples.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { MockSettings } from '../../../../stories/helperComponents/MockSettings';
+import CodeExample from 'wix-storybook-utils/CodeExample';
+
+import * as ExtendedRawSource from '!raw-loader!./LikeButtonExtendedExample.tsx';
+import * as ExtendedCSSRawSource from '!raw-loader!./LikeButtonExtendedExample.st.css';
+import { LikeButtonExtendedExample } from './LikeButtonExtendedExample';
+
+export default () => (
+  <div>
+      <div className="tpa-container">
+      <CodeExample
+        title="Like Button"
+        code={[
+          '//.st.css',
+          ExtendedCSSRawSource,
+          '',
+          '//.tsx',
+          ExtendedRawSource,
+        ].join('\n')}
+      >
+        <LikeButtonExtendedExample 
+          label="Like" 
+          labelPlacement="right" 
+          checked={false} 
+          disabled={false} />
+      </CodeExample>
+    </div>
+    <MockSettings
+      wixColorParams={[
+        {
+          label: 'Icon Color',
+          wixParam: 'iconColor',
+          defaultColor: 'color-5',
+        },
+      ]}
```

Comment:
We should probably also add MockSettings for the label color, and font.
I think you can look at the Button's example to see how they added multiple mocked settings

Reply:
Done